### PR TITLE
Enable to test blueoil part by each tasks

### PR DESCRIPTION
**Pull request by hadusam**
Enable to specify test of blueoil part by each task for #355

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature / Optimization (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by iizukak** [comment]
@hadusam 
When execute
```
$ make test-object_detection
```
then I got this error message

https://gist.github.com/iizukak/d429228db4803e6e70e38d0906f36261

File: Makefile [hunk 1]
```diff
@@ -15,9 +15,22 @@ build: deps
...
-	# Run Blueoil test
-	CUDA_VISIBLE_DEVICES=$(CUDA_VISIBLE_DEVICES) bash ./blueoil_test.sh
+test: build test-classification test-object_detection test-semantic_segmentation
+
+.PHONY: test-classification
+test-classification: build
+	# Run Blueoil test of classification
+	CUDA_VISIBLE_DEVICES=$(CUDA_VISIBLE_DEVICES) bash ./blueoil_test.sh  --task classification
+
+.PHONY: test-object_detection
```
Comment (iizukak) [inline Makefile]: In `test-object_detection`, mixed "-" and "_" in one word looks not good.

Comment (hadusam) [reply Makefile]: Thank you. I fixed to use `-` instead of `_`. :+1:

**Comment by hadusam** [comment]
@iizukak 
Thanks for tests. But I cannot reproduce your error on my environment.
And according to the current error log setting, I cannot inspect what happened.
I update this branch by merging current master (including #341), so tests will pass stably or output more information about your error.
Could you please re-try tests?:bow: Thank you.

**Comment by iizukak** [comment]
@hadusam Thanks. I will retest.

**Comment by iizukak** [comment]
@hadusam Test passed 🤓

**Comment by hadusam** [comment]
@iizukak 
Oh... I found these error, there may be something wrong in copying process...(scp?) :thinking:
> ImportError: No module named nnlib
> python: can’t open file ‘testing_code.py’: [Errno 2] No such file or directory
> OSError: ./lib_arm_fpga.so: cannot open shared object file: No such file or directory

I will retry to test.

**Comment by hadusam** [comment]
run dlk test

**Pull request merged**